shared/productTypes.ts

```diff
@@ -579,6 +581,54 @@ export const PRODUCT_TYPES: Record<ProductTypeKeys, ProductType> = {
 			swapFeedbackAndContactUs: true,
 		},
 	},
+	voucherobserver: {
+		productTitle: () => 'Newspaper Voucher - Observer',
+		friendlyName: 'newspaper voucher subscription - Observer',
+		shortFriendlyName: 'newspaper voucher booklet',
+		productType: 'voucher',
+		groupedProductType: 'subscriptions',
+		allProductsProductTypeFilterString: 'Voucher',
+		urlPart: 'voucher',
+		checkoutUrlPart: '/subscribe', // https://support.theguardian.com/uk/subscribe
+		getOphanProductType: () => 'PRINT_SUBSCRIPTION',
+		productPageNewsletterIDs: [FRONT_PAGE_NEWSLETTER_ID],
+		softOptInIDs: [
+			SoftOptInIDs.SupportOnboarding,
+			SoftOptInIDs.SubscriberPreview,
+			SoftOptInIDs.SupporterNewsletter,
```

Comment:
is this right, I thought soft opt ins were completely off for observer only?

Reply:
https://github.com/guardian/support-service-lambdas/blob/14df86afac680264a811409ce93f854835bc6b94/handlers/soft-opt-in-consent-setter/src/main/scala/com/gu/soft_opt_in_consent_setter/models/ConsentsMapping.scala#L64

Reply:
I'm going to remove these options from all of the Observer products in a subsequent pr 👍